etherpad-centos7-atomicapp/artifacts/kubernetes/etherpad-rc.yaml

```diff
@@ -28,6 +28,6 @@ spec:
             - name: DB_PASS
               value: $db_pass
             - name: DB_PORT
-              value: '3306'
+              value: $db_port
```

Comment:
this fix killed the kubernetes support for etherpad :( https://ci.centos.org/job/atomicapp-test-kubernetes-pr/141/console